recipes/gumbo-parser/all/conanfile.py

```diff
@@ -0,0 +1,67 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class GumboParserConan(ConanFile):
+    name = "gumbo-parser"
+    description = "An HTML5 parsing library in pure C99"
+    topics = ("conan", "parser")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/google/gumbo-parser"
+    license = "Apache-2.0"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+        if self.settings.compiler == "Visual Studio":
+            raise ConanInvalidConfiguration("%s is not supported by Visual Studio" % self.name)
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_folder = "gumbo-parser-{0}".format(self.version)
+        os.rename(extracted_folder, self._source_subfolder)
+
+    def _configure_autotools(self):
+        if not self._autotools:
+            with tools.chdir(self._source_subfolder):
+                self.run("./autogen.sh", win_bash=tools.os_info.is_windows)
+            self._autotools = AutoToolsBuildEnvironment(self, win_bash=tools.os_info.is_windows)
+            args = []
+            if self.options.shared:
+                args.extend(['--disable-static', '--enable-shared'])
+            else:
+                args.extend(['--disable-shared', '--enable-static'])
+            self._autotools.configure(configure_dir=self._source_subfolder, args=args)
+        return self._autotools
+
+    def build(self):
+        autotools = self._configure_autotools()
+        autotools.make()
+
+    def package(self):
+        self.copy(pattern="COPYING", dst="licenses", src=self._source_subfolder)
+        autotools = self._configure_autotools()
+        autotools.install()
+        tools.rmdir(os.path.join(self.package_folder, 'lib', 'pkgconfig'))
+        la_file = os.path.join(self.package_folder, 'lib', 'libgumbo.la')
+        if os.path.isfile(la_file):
+            os.unlink(la_file)
```

Comment:
Is the test necessary?
```suggestion
        os.unlink(os.path.join(self.package_folder, 'lib', 'libgumbo.la'))
```

Reply:
Yes, because la file can be missing (for example, non-unix builds). checking for la_file is a common practice in recipes

Reply:
Not when a probject is using libtool.
- https://github.com/google/gumbo-parser/blob/aa91b27b02c0c80c482e24348a457ed7c3c088e0/configure.ac#L33
- https://github.com/google/gumbo-parser/blob/master/Makefile.am#L44

Since you are using `autoreconf` + `./configure` + `make`, you are also using it.